src/components/playground-container.jsx

```diff
@@ -28,10 +28,10 @@ export default class PlaygroundContainer extends React.Component {
   }
 
   render() {
-    const {scope, source, noRender, playgroundtheme, exportGist} = this.props;
+    const {scope, source, noRender, playgroundtheme, exportGist, copyToClipboard} = this.props;
     return (
       <div className="Interactive">
-        {exportGist ? this.renderToolbar() : null}
+        {exportGist || copyToClipboard ? this.renderToolbar() : null}
```

Comment:
Just call `this.renderToolbar()` here, and return `null` from that function if the prerequisites aren't met.